display_list/dl_builder.cc

```diff
@@ -437,7 +438,7 @@ void DisplayListBuilder::saveLayer(const DlRect& bounds,
 
   // A backdrop will affect up to the entire surface, bounded by the clip
   bool will_be_unbounded = (backdrop != nullptr);
-  std::shared_ptr<const DlImageFilter> filter;
+  std::shared_ptr<DlImageFilter> filter;
```

Comment:
Why change this?

Reply:
The classes are immutable and I was having trouble with the fact that you can't mix and match shared_ptr with different "const" modifiers.

Reply:
https://google.github.io/styleguide/cppguide.html#Use_of_const

Immutable classes should have const methods so having a const reference should still be the appropriate usage in c++.